src/games/PlayerVsPlayerGame.js

```diff
@@ -23,4 +31,44 @@ PlayerVsPlayerGame.prototype.switchMarker = function(playerMarker) {
   return playerMarker;
 }
 
+PlayerVsPlayerGame.prototype.endGame = function(gameBoard) {
```

Comment:
I think the PlayerVsPlayerGame may be getting a little 'too smart' knowing about who's turn it is. This is all fine for now, but there may need to be a some sort of `board` abstraction in the near future. (Also it would be cool to see the winning combinations created dynamically 😎 + you could still use all the same tests!)